universal_transfer_operator/pyproject.toml

```diff
@@ -0,0 +1,149 @@
+[build-system]
+requires = ["flit_core ~=3.2"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "universal-transfer-operator"
+dynamic = ["version"]
+description = """
+"""
+
+authors = [
+    { name = "Astronomer", email = "humans@astronomer.io" },
+]
+readme = "README.md"
+license = { file = "LICENSE" }
+
+requires-python = ">=3.7"
+dependencies = [
+    "apache-airflow>=2.0",
+    "attrs>=20.0",
+    "pandas>=1.3.4,<2.0.0", # Pinning it to <2.0.0 to avoid breaking changes
+    "pyarrow",
+    "python-frontmatter",
+    "smart-open",
+    "SQLAlchemy>=1.3.18",
+    "apache-airflow-providers-common-sql"
+]
+
+keywords = ["airflow", "provider", "astronomer", "sql", "decorator", "task flow", "elt", "etl", "dag"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: Apache Software License",
+    "Topic :: Database",
+    "Framework :: Apache Airflow",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+]
+
+[project.optional-dependencies]
+tests = [
+    "pytest>=6.0",
+    "pytest-split",
+    "pytest-dotenv",
+    "requests-mock",
+    "pytest-cov",
+    "pytest-describe",
+    "types-requests",
+    "mypy",
+    "sqlalchemy-stubs", # Change when sqlalchemy is upgraded https://docs.sqlalchemy.org/en/14/orm/extensions/mypy.html
```

Comment:
Do we need all of this?

Reply:
[Removed](https://github.com/astronomer/astro-sdk/pull/1492/commits/c2e08eaec89c463d8e83e7a8fa523231e56ede4f)